api/migrations/20241010123909-updateinscriptionStepForYoungCleBug.js

```diff
@@ -0,0 +1,53 @@
+const { logger } = require("../src/logger");
+const { YoungModel } = require("../src/models");
+
+module.exports = {
+  async up(db, client) {
+    try {
+      // Trouver les jeunes avec les critères spécifiques
+      const youngs = await YoungModel.find({
+        status: "IN_PROGRESS",
+        source: "CLE",
+        inscriptionStep2023: "DOCUMENTS",
+      }).select({ _id: 1 });
+
+      logger.info(`Found ${youngs.length} youngs to update.`);
+
+      const youngIds = youngs.map(({ _id }) => _id);
+
+      // Mettre à jour les jeunes trouvés pour changer leur `inscriptionStep2023` à "REPRESENTANTS"
+      await YoungModel.updateMany(
```

Comment:
Eviter les updateMany (on n'a pas de traça). Itérer sur les users et faire un .save() avec le nom de la migration dans le fromUser.